console.rb

```diff
@@ -4,14 +4,93 @@
 
 # NOTE: Uncomment the lines below to verify that your seed script is working
 
+#
+#
 # puts "There are #{Apartment.count} apartments"
 # puts "There are #{Tenant.count} tenants"
 #
 # puts "*" * 50
 #
 # puts "The first apartment is at #{Apartment.first.address}."
 # puts "It has  #{Apartment.first.tenants.count} tenants."
+#
+# binding.pry
+#
+# puts "ignore this line, it's just here so the binding.pry above works"
+
+puts "
+          Welcome to Tenant / Apartment Management System
+          Input following commands to proceed
+          "
+      userInput=nil
+
+
+      userInput = 'menu'
+              while !(userInput==nil)
+                case userInput
```

Comment:
Watch you indentation, a best practice is to be consistent with how much each level is indented - 2 spaces is most common, 4 spaces is less common, 3 spaces is generally bad.